more-info pages for unsupported systems (#15229)
home-assistant/home-assistant.github.io: source/more-info/unsupported/dbus.markdown

---
title: "D-Bus issues"
description: "More information on why missing D-Bus marks the installation as unsupported."
---

## The issue

> D-Bus is at how the Supervisor can do most of the communication with the host,

Reviewer: ```suggestion
D-Bus is how the Supervisor does most of the communication with the host,
```
